distributed/tests/test_cancelled_state.py

```diff
@@ -535,3 +536,146 @@ async def release_all_futures():
                 (f3.key, "resumed", "error", "error", {}),
             ],
         )
+
+
+[
```

Comment:
What's this for?

Reply:
I thought flake8 detected unused expressions like this

Reply:
Me too, let me check our `flake8`.

Reply:
Ah this is in flake8-bugbear B018

Reply:
oops. that was me debugging something